tests/Unit/Infrastructure/Mail/ErrorHandlingTemplateBasedMailerTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Tests\Unit\Infrastructure\Mail;
+
+use PHPUnit\Framework\TestCase;
+use Psr\Log\LogLevel;
+use WMDE\EmailAddress\EmailAddress;
+use WMDE\Fundraising\Frontend\Infrastructure\Mail\ErrorHandlingTemplateBasedMailer;
+use WMDE\Fundraising\Frontend\Tests\Fixtures\ErrorThrowingTemplateBasedMailer;
+use WMDE\Fundraising\Frontend\Tests\Fixtures\TemplateBasedMailerSpy;
+use WMDE\PsrLogTestDoubles\LoggerSpy;
+
+/**
+ * @covers \WMDE\Fundraising\Frontend\Infrastructure\Mail\ErrorHandlingTemplateBasedMailer
+ */
+class ErrorHandlingTemplateBasedMailerTest extends TestCase {
+
+	public function testOnSendMail_sendsMail(): void {
+		$mailerSpy = new TemplateBasedMailerSpy( $this );
+		$email = new EmailAddress( 'happy@bunny.carrot' );
+		$arguments = [ 'gotta get up' => 'to get down' ];
+
+		$errorHandlingMailer = new ErrorHandlingTemplateBasedMailer(
+			$mailerSpy,
+			new LoggerSpy()
+		);
+
+		$errorHandlingMailer->sendMail( $email, $arguments );
+
+		$mailerSpy->assertCalledOnceWith( $email, $arguments );
```

Comment:
You could have checked that the logger did not get any calls, but that's not a blocker.